PROPOSAL_FOR_RESPONSE_DESCRIPTIONS.md

```diff
@@ -0,0 +1,244 @@
+# Proposal for supporting response descriptions in Apipie
```

Comment:
Since we are getting close to merging this, would you mind turning this into the documentation inside README.rst?

Reply:
Sure thing.

Reply:
I pushed a version now with most of the README.rst work.  Still need to document the `response description adapter` mechanism, but the straightforward use of the `returns` and `property` keywords is now there.  Would you mind taking a look and letting me know if you have any comments?